cla_frontend/templates/base.html

```diff
@@ -129,6 +129,9 @@ <h1>:(</h1>
     </main>
 
     {% block base_javascript %}
+    <script type="text/javascript">
+      window.family_issue_flag = "{{ family_issue_flag }}" === "True";
```

Comment:
This looks wrong to me. Your setting does a comparison to the string "True", so the setting is a boolean. So comparing the setting to the string "True" will always be false, no?

Have you tested this code?

Reply:
Yeah there's quotes around the `family_issue_flag` so whatever we get from django will always be in a string

Reply:
Oh ok I see. Hmm yeah maybe there's not a neater way of doing this step.
I'm not sure why we're bothering turning the env var into a Python `True` then, that's just misleading and confusing. What do you think to changing the Python variable to just be the string from the env var?

Reply:
Yeah I think's that a good idea since we have to do the comparison in JS, so having it in string form makes sense.

Reply:
Ah ok I realised as well the reason behind the quotes `"{{ family_issue_flag }}"` is because without it, JS thinks this is a variable and it will error. 

I was thinking we could do `{{ family_issue_flag }} === "True"`, but that won't work

Reply:
I think the issue is when you do `FAMILY_ISSUE_FEATURE_FLAG = os.environ.get("FAMILY_ISSUE_FEATURE_FLAG", "False") == "True"`  then `FAMILY_ISSUE_FEATURE_FLAG` will contain a boolean value

Then when you do `window.family_issue_flag = "{{ family_issue_flag }}" === "True";` you are turning boolean value back into a string. So Michael comment is suggesting why not just keep`FAMILY_ISSUE_FEATURE_FLAG` a string in your settings file

If you do that then you need to update `<div ng-if="family_issue_flag">` to be something like `<div ng-if="family_issue_flag == 'True'">`

Reply:
I can change `FAMILY_ISSUE_FEATURE_FLAG` to a string, but I still need to wrap the variable `{{ family_issue_flag }}` in a string because without it I will get the error I showed above. Whatever `FAMILY_ISSUE_FEATURE_FLAG` is, JS seems to interpret as a variable. 

Seeing as I will have to wrap it in quotes anyway, is there still need to `FAMILY_ISSUE_FEATURE_FLAG` to a string?

Reply:
Yes, the reason is that having it be a Python boolean is just confusing if it's never going to actually be used as one.

I don't think you would need to change the js snippet at all.

And the reason it's not working in js without the quote marks is because js booleans are lowercase, ie `true` in this case. If you don't make it a string literal in js, with the quote marks, it needs to be a valid something else. If you did `foo = False` in your console you wouldn't expect it to work, but you would expect `foo = false` to work.

So you could make the Python string be `"true"` or `"false"` and then use that as a boolean literal in the js, but I'm not sure I like that either: the fact that the casing of the string is important for the js is not obvious when it's being set in the Python layer, so I'd like to avoid it as that's just as confusing as setting a Python boolean that only ever gets used as a string.

Reply:
Ok I think I understand. It's about the representation of the feature flag, not the implementation. I was thinking it wouldn't matter because the implementation would be the same regardless.

But from what your saying it's about the fact it's never going to be used as a boolean, so having it set that way is confusing. Gotcha

Reply:
Done